src/main/java/uk/gov/hmcts/reform/cwrdapi/repository/CaseWorkerProfileRepository.java

```diff
@@ -24,6 +24,11 @@ public interface CaseWorkerProfileRepository extends JpaRepository<CaseWorkerPro
 
     List<CaseWorkerProfile> findByEmailIdIgnoreCaseContaining(String emailPattern);
 
+    @Query(value = "select cw from case_worker_profile cw where lower(cw.firstName) like "
+            + "concat('%', :searchString, '%') "
```

Comment:
can this query support full first name or last name search ? and how the records will be order by last name , first name asc ?

Reply:
Its done in page at controller